### DISCUSSION TITLE
Network Throttling

### DISCUSSION THREAD

**Discussion by jianshangwei94** [outline/outline#4618] [Ideas]
Do anyone has the way to limit the network bandwidth? for each access key?

I tried on Wireguard app with wondershare It could be limited in applicaiton throttling but there is no way to have individualy to limit the bandwith speed.

I suggest that it'd be more better that's comes with Data limitation and additionalionally with bandwidth limitation

**Comment by apoorv-mishra** [comment]
I believe you're misdirected here. You're looking for a different product(with identical name :laughing: ). Hope [this helps](https://github.com/outline/outline/discussions/4576#discussioncomment-4402547).
